tabbing/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
I wonder if this script should live [inline in the head](https://github.com/Financial-Times/n-ui/blob/7fbadc0e8a4cff719b961fba924a7f4f0e4ad662/layout/wrapper.html#L64) rather than be loaded externally. Then JS people on slow/flaky connections won’t get a Moment of Rubbishly-Focussed Links (MORFL 😂 )